### ISSUE TITLE
[13.1.0 OSX] Beachball downloading the movie list

### ISSUE THREAD

**Issue by Jako** [mediathekview/MediathekView#332]
The latest version of MediathekView (13.1.0) seems to have issues on OSX while downloading the movie list. The download is very slow and the progress bar stops to get further around 80%. 

Could I debug somehow what's going wrong?

**Comment by zxsd** [comment]
In the future, the [Forum](https://forum.mediathekview.de/recent) is likely a good starting point with problems like this.  Here are a couple of links that appear to be relevant:

[MV 13.1.0 Startproblem](https://forum.mediathekview.de/topic/1547/mv-13-1-0-startproblem)

[Sachstand zu den Problemen von 13.1](https://forum.mediathekview.de/topic/1571/sachstand-zu-den-problemen-von-13-1)

I strongly believe when @derreisende77 releases version 13.1.1, problems such as you are experiencing will also be addressed.  I'm a Windows user, but have read in the Forum that Mac OSX users should not implement the ZIP version, rather use the [DMG](https://mediathekview.de/download/) version.

**Comment by Spoekenkieker** [comment]
Just FYI, both:

For me it also didn’t stop loading the movie list, so I killed and relaunched it, this time it DID stop but then : spinning beachball of death … see [my forum post](https://forum.mediathekview.de/topic/1577/hilfe-v-13-1-0-macos-h%C3%B6rt-nach-start-nicht-auf-filmliste-zu-laden).

And @zxsd: happened with the DMG version—macOS version comes on the DMG.

For now I rolled back to 13.0.6.
